package.json

```diff
@@ -19,7 +19,8 @@
   },
   "dependencies": {
     "@agoric/casting": "^0.4.2",
-    "@agoric/cosmic-proto": "~0.3.0",
+    "@agoric/client-utils": "0.1.1-dev-280e215.0",
```

Comment:
This is mixing `dev` tags with SDK release (`u20`) tags with version numbers. It risks having three versions of each package. That could balloon the build but this app is going to be barely used so I think it's ok. cc @samsiegart in case in disagrees. I think it's safe to merge this though and that cleanup could be followup.

Reply:
> cleanup could be followup.

Thanks!  That helps me limit the scope of this PR only to remove the `/custom/` query.